lib/services/components.js

```diff
@@ -469,8 +473,12 @@ function getStyles(name, slug) {
 function getScripts(name, slug) {
   const site = siteService.sites()[slug],
     assetDir = site && site.assetDir || mediaDir,
-    assetDirLen = assetDir.length,
-    assetPath = site && site.assetPath || '';
+    assetDirLen = assetDir.length;
+  let assetPath = site && site.assetPath || '';
+
+  if (site && site.path && site.assetPath !== site.path) {
+    assetPath = site.path;
+  }
```

Comment:
Yeah, agree it is all a bit awkward. I just kept tweaking it to meet the tests which I could not get to pass with config changes alone.  These two tests are what are needed to run all current sites.

I think there are 2 real-world cases (and 1 not so real, but possible via current config options):
1) 
site.path = '/scienceofus'
site.assetPath = '/scienceofus'
sample url: /scienceofus/css/public-login.css'
filepath: '/css/simple-header.scienceofus.css'
2) 
site.path = '/'
site.assetPath = '/vulture'
sample url: /css/simple-header.css'
filepath: '/css/simple-header.vulture.css'
3) 
site.path = '/scienceofus'
site.assetPath = '/scienceofusAssetPath'
sample url: /scienceofus/css/public-login.css'
filepath: '/css/simple-header.scienceofus.css' ????